### PR TITLE
MULTIARCH-4545: :warning: Implement multi-version API and promote current v1alpha1 to v1beta1

### PR DESCRIPTION
**Pull request by AnnaZivkovic**
Move CR Apiversion from v1alpha1 to v1beta1. 

Ran  operator-sdk create api --group multiarch --version v1beta1 --kind PodPlacementConfig
and implemented conversions. Still working on how to implement the webhooks. 

Following this tutorial 
https://book.kubebuilder.io/multiversion-tutorial/tutorial

### CODE REVIEW

File: PROJECT [hunk 1]
```diff
@@ -19,11 +19,18 @@ resources:
   version: v1
 - api:
     crdVersion: v1
-    namespaced: false
```
Comment (AnnaZivkovic) [inline PROJECT]: Keep namespaced: false

**Comment by Prashanth684** [comment]
I am not very familar with api conversions - but aren't conversions needed if we have shipped a release? since we have no prior release, can't we just straight up change the APIs to v1beta1 ?

**Comment by Prashanth684** [comment]
> I am not very familar with api conversions - but aren't conversions needed if we have shipped a release? since we have no prior release, can't we just straight up change the APIs to v1beta1 ?

just discussed with @aleskandro and let's stick to this for now. @AnnaZivkovic how much effort is the webhook conversion?

**Comment by aleskandro** [comment]
/lgtm
/approve

:tada:

**Pull request merged**